### PR TITLE
docs: Clay CSS 2 Recipes Getting Started Source formatting; Word smith; Update grammar

### PR DESCRIPTION
**Pull request by pat270**

### CODE REVIEW

**Comment by wincent** [comment]
The Semantic Pull Request bot is complaining because both PR and commit subject line need to start with a "type" (eg. "docs:", or you could use "style:" if it were just style changes).

Prettier in CI seems to disagree with formatting in css/clay_css_2_recipes/01_getting_started.md — did you run `yarn format` locally? I don't think we've change the version of Prettier here recently ([last was April 6th](
https://github.com/liferay/liferay-frontend-guidelines/commit/77ff6323345a2bb350ea472687d3fb23d89fb333)), so I doubt you'd need to run `yarn` to update it.

**Comment by wincent** [comment]
I think we can ignore the failures from the `checkLinks` script:

```
dxp/how_to_use_css_custom_properties.md: Status code 503 for remote link: https://codepen.io/eduardoallegrini/pen/WNvgWMp
dxp/how_to_use_css_custom_properties.md: Status code 503 for remote link: https://codepen.io/eduardoallegrini/pen/gOpdyKP
dxp/how_to_use_css_custom_properties.md: Status code 503 for remote link: https://codepen.io/eduardoallegrini/pen/vYEGYZK
```

They work for me now with a manual check.

File: css/clay_css_2_recipes/01_getting_started.md [hunk 1]
```diff
@@ -223,53 +233,55 @@ Clay CSS uses `rem` based values instead of `px` for accessibility. This ensures
...
 
-```
+```scss
 $headings-color: $gray-900;
 $headings-font-weight: 400;
 $headings-margin-bottom: 0.5rem;
 ```
 
-We don't need to change:
+We can keep the following variables set to their default values, since the `family-family` is already set on the body and `$headings-line-height` is already set to 1.2.
```
Comment (wincent) [inline css/clay_css_2_recipes/01_getting_started.md:260]: `family-family`? Should that be `font-family`?

Comment (pat270) [reply css/clay_css_2_recipes/01_getting_started.md]: most definitely, pushed new changes. Thanks @wincent!

**Comment by wincent** [comment]
lg2m: `checkLinks` failed again. If it keeps happening we can tweak the script (we had to do that [once in the past](https://github.com/liferay/liferay-frontend-guidelines/pull/134) because CodePen is a beautiful and unique snowflake), but I don't want to do anything until it's clear that this is a non-transient problem (CI was [fine last week](https://github.com/liferay/liferay-frontend-guidelines/runs/671992960)).

**Pull request merged**